### ISSUE TITLE
Bug? if not, Feature Request: if {} else if {} code collapse in editor

### ISSUE THREAD

**Issue by MichaelAtOz** [openscad/openscad#1392]
See below.

The red highlight shows that the `if `and `else` parts can be collapsed separately.
The blue, using `else if` cannot, it collapses the whole statement. Not so handy if it has many `if else`'s and is 4 pages long.

![if else collapse](https://cloud.githubusercontent.com/assets/1972961/8763873/0f8f0a7e-2df6-11e5-9871-758a24eb214d.JPG)

(code for cut/paste)
```
// if & else if

if (true)
	cube(1);
else
	cube(2);

if (false) {
	cube(3);
}


// syntax break
if (true)
	{
	cube(4);
	cube(5);
	}
else
	{
	cube(6);

	}


// syntax break
if (true) {
	cube(7);
	//
} else if {
	cube(8);
	//
} else if {
	cube(9);
	//
} else if {
	cube(10);
	//
}

// syntax break
```

**Comment by MichaelAtOz** [comment]
This is what it looks like in Notepad++

![if else collapse notepad](https://cloud.githubusercontent.com/assets/1972961/8768872/12c2e314-2ed2-11e5-84ac-288dc9353575.JPG)

**Comment by kintel** [comment]
Fixed by #1776
